fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/event/Event.java

```diff
@@ -28,4 +32,16 @@ public int compareTo(Event o) {
 
     return Long.compare(o.getEndTimestamp(), this.getEndTimestamp());
   }
+
+  /**
+   * @return true if event the event takes place today
+   */
+  public boolean isEventToday() {
+    long currentTime = System.currentTimeMillis();
+    return getEndTimestampInMillis() - currentTime < Constants.MS_IN_A_DAY;
```

Comment:
Isn't this really "isEventEndingToday" ? What about events that could span multiple days (like an election could) ?

Reply:
I Hadn't thought about that. It's fixed 👍